### PR TITLE
Fix hardcoded value in FormatCodeCommand.cs

### PR DESCRIPTION
**Pull request by Gulmorg**

### CODE REVIEW

File: src/Core/Models/Config/BotConfiguration.cs [hunk 1]
```diff
@@ -127,6 +127,10 @@ public class BotConfiguration
         [Description("The minimum time between being able to use the latex command again, in milliseconds")]
         public int LatexTimeoutMillis { get; set; }
 
+        [YamlMember(Alias = "guruDeleteTresholdMillis")]
+        [Description("The maximum time before a message can be deleted when the format code command is used by a guru, in milliseconds")]
```
Comment (AquaXV) [inline src/Core/Models/Config/BotConfiguration.cs:131]: command is used by Guru and up, and is not a guru specific requirement. Change name so it describes function

File: src/Commands/Modules/Configuration/ConfigurationCommand.cs [hunk 2]
```diff
@@ -1,32 +1,97 @@
...
+using Discord.WebSocket;
 
 using BrackeysBot.Services;
 
 namespace BrackeysBot.Commands
 {
     public partial class ConfigurationModule : BrackeysBotModule
     {
-        public ConfigurationService Config { get; set; }
+        private IUserMessage _lastConfigMessage;
```
Comment (AquaXV) [inline src/Commands/Modules/Configuration/ConfigurationCommand.cs:15]: So what happens if 2 people use []config in different channels?

**Review by AquaXV — CHANGES_REQUESTED** [review]

File: src/Commands/Modules/Configuration/ConfigurationCommand.cs (hunk 2 above)
Comment (Gulmorg) [reply src/Commands/Modules/Configuration/ConfigurationCommand.cs:15]: It still works individually for each embed as it subscribes to different versions of the event.

File: src/Core/Models/Config/BotConfiguration.cs (hunk 1 above)
Comment (Gulmorg) [reply src/Core/Models/Config/BotConfiguration.cs:131]: Changed it to CodeFormatterDeleteTresholdMillis

**Review by AquaXV — APPROVED** [review]

**Pull request merged**